autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/service/RealAutofillService.kt

```diff
@@ -100,7 +100,9 @@ class RealAutofillService : AutofillService() {
 
                 callback.onSuccess(response)
             }.onFailure {
-                pixel.fire(AutofillPixelNames.AUTOFILL_SERVICE_CRASH, mapOf("message" to it.extractExceptionCause()))
+                if (it !is kotlinx.coroutines.CancellationException) {
```

Comment:
While testing I've noticed some crash pixels related to the coroutine being cancelled. That doesn't represent a crash, so skipping if that's the case.